File: ESC.Resturant.Data/Entities/Taxes.cs
using ESC.Resturant.Data.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public class Taxes
    {
        [Key]
        public int Id { get; set; }
        public string NameAr { get; set; }
        public string NameEn { get; set; }
        public double PercentValue { get; set; }
        public Nullable<int> DiscountId { get; set; }
        public bool IsActive { get; set; }
        public bool IsDelete { get; set; }
        public virtual ICollection<BillTaxes> BillTaxes { get; set; }
        public virtual DiscountType DiscountType { get; set; }

    }
}

File: ESAResturant.NEW.web/ClientApp/src/app/admin/setting/Models/Taxes.ts

export class Taxes {

  constructor() {

    this.id = 0;
    this.nameAr = '';
    this.nameEn = '';
    this.percentValue = 0;
    this.isActive = false;
    this.isDelete = false;
    this.discountId = 1;

  }

  id: number;
  nameAr: string;
  nameEn: string;
  percentValue: number;
  isActive: boolean;
  isDelete: boolean;
  discountId: number;

}
